libbpf-tools/funclatency.c

```diff
@@ -0,0 +1,335 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+/* Copyright (c) 2021 Google LLC.
+ *
+ * Based on funclatency from BCC by Brendan Gregg and others
+ * 2021-02-26   Barret Rhoden   Created this.
+ *
+ * TODO:
+ * - support uprobes on libraries without -p PID. (parse ld.so.cache)
```

Comment:
looking briefly at command line args of funclatency.py, it seems like there are more features that are not supported (e.g., regexp pattern matching and per-function historgrams). It's fine, but let's document all the lacking features somewhere here?